app/component/AddFavouriteContainer.js

```diff
@@ -18,6 +18,7 @@ class AddFavouriteContainer extends React.Component {
     intl: intlShape.isRequired,
     executeAction: PropTypes.func.isRequired,
     router: PropTypes.object.isRequired,
+    location: PropTypes.object.isRequired,
```

Comment:
can we use https://github.com/ReactTraining/react-router/blob/v2.8.1/docs/API.md#proptypes for these?